Website/Portals/_default/HotcakesViews/_default/Scripts/StripeCheckout.js

```diff
@@ -0,0 +1,116 @@
+// This is a public sample test API key.
+// Don’t submit any personally identifiable information in requests made with this key.
+// Sign in to see your own test API key embedded in code samples.
+const stripe = Stripe("pk_test_TYooMQauvdEDq54NiTphI7jx");
+
+// The items the customer wants to buy
+const items = [{ id: "xl-tshirt" }];
+
+let elements;
+
+initialize();
+checkStatus();
+
+document
+  .querySelector("#Form")
+  .addEventListener("submit", handleSubmit);
+
+// Fetches a payment intent and captures the client secret
+async function initialize() {
+    const response = await fetch("/DesktopModules/Hotcakes/API/mvc/PaymentIntentApi/CreatePaymentIntent", {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify({ items }),
+  });
+  const { clientSecret } = await response.json();
+
+  const appearance = {
+    theme: 'stripe',
```

Comment:
What are the options for this?  Are there others that come with Stripe, or would we have to build out a template/design/theme?